src/pykeen/training/slcwa.py

```diff
@@ -57,6 +57,7 @@ def _create_training_data_loader(
             dataset=triples_factory.create_slcwa_instances(
                 batch_size=batch_size,
                 shuffle=kwargs.pop("shuffle", True),
+                instance_weighting="test",
```

Comment:
by the way, I assume the weighting procedure should be passed via `train` (i.e. via `training_kwargs`) and not via `training_loop_kwargs` because the same `TrainingLoop` may be used multiple times with different weights. Is that right?